contracts/0.4.24/Lido.sol

```diff
@@ -559,20 +664,34 @@ contract Lido is ILido, IsContract, StETH, AragonApp {
     * @return amount of StETH shares generated
     */
     function _submit(address _referral) internal whenNotStopped returns (uint256) {
-        address sender = msg.sender;
-        uint256 deposit = msg.value;
-        require(deposit != 0, "ZERO_DEPOSIT");
+        uint256 stakeLimitSlotValue = STAKE_LIMIT_POSITION.getStorageUint256();
+
+        require(!stakeLimitSlotValue.isStakingPaused(), "STAKING_PAUSED");
+        require(msg.value != 0, "ZERO_DEPOSIT");
```

Comment:
Maybe move this to the top to fails earlier in the case?

Reply:
Done, thx!